components/atom/button/src/index.scss

```diff
@@ -27,6 +29,39 @@ $icon-sz-atom-button-large: $sz-icon-m !default;
 $icon-m-atom-button-large: $m-m !default;
 $p-atom-button-large: $p-l !default;
 
+// Primary
+$c-atom-button-primary: $c-white !default;
+$bgc-atom-button-primary-focus: $bgc-atom-button-dark !default;
+$bdc-atom-button-primary-focus: $bgc-atom-button-dark !default;
+$bgc-atom-button-primary: $bgc-atom-button !default;
+$bdc-atom-button-primary: $bgc-atom-button !default;
+
+// Secondary
+$bgc-atom-button-secondary: $bgc-atom-button !default;
+$bgc-atom-button-secondary-focus: $bgc-atom-button-light !default;
+
+$bdrs-atom-buton-first-secondary: $bdrs-m 0 0 $bdrs-m !default;
+$bdrs-atom-buton-last-secondary: 0 $bdrs-m $bdrs-m 0 !default;
+
+// Tertiary
+$bdc-atom-button-tertiary: transparent !default;
+
+// Negative
+$c-atom-button-negative-primary: $bgc-atom-button !default;
+$bgc-atom-button-negative-primary-focus: $bgc-atom-button-light-contrast !default;
+$bdc-atom-button-negative-primary-focus: $bgc-atom-button-light-contrast !default;
+$bgc-atom-button-negative-primary: $c-white !default;
+$bdc-atom-button-negative-primary: $c-white !default;
+$c-atom-button-negative-secondary: $c-white !default;
+$bgc-atom-button-negative-secondary-focus: $bgc-atom-button-dark !default;
+
+// Accent
+$c-atom-button-accent: $c-white !default;
+$bgc-atom-button-accent-focus: $c-accent-dark !default;
+$bdc-atom-button-accent-focus: $c-accent-dark !default;
+$bgc-atom-button-accent: $c-accent !default;
+$bdc-atom-button-accent: $c-accent !default;
```

Comment:
Should they go in a `_settings.scss` file?

Reply:
Good question, although I don't see any _settings.scss file in any SUI component. All of them have the var definitions inside the component file.

Reply:
There are some with the _settings.scss file, I guess it is optional